source/manual/github-unavailable.html.md

```diff
@@ -6,62 +6,53 @@ layout: manual_layout
+ECR_REGISTRY="172025368201.dkr.ecr.eu-west-1.amazonaws.com"
+REPO=$(awk -F '=' /GOVUK_APP_NAME/'{print $2}' Dockerfile)
+```
 
-Once you've [pushed your hotfix branch to CodeCommit](#making-changes-to-code-in-aws-codecommit-before-deployment), you should:
+1. Build the docker image locally with the tag named after the latest sha.
 
-1. Create a pull request in the CodeCommit UI.
-1. Ask another developer to review it as a sense check.
-1. With the branch reviewed, you can then [deploy it](#deploy-from-aws-codecommit), specifying your branch name as the `TAG` (in the example above, 'hotfix'). Do this in Integration or Staging first, then manually test the changes.
-  - In the example above, you can check the README was removed by SSH'ing into the relevant machine and executing `ls /var/apps/{REPO_NAME}` to confirm that the README is absent.
-1. Once you're confident in your changes, deploy the branch to Production.
-1. Record the missing deployment in the Release app.
+```
+ docker build --platform linux/amd64 -f Dockerfile -t $IMAGE_TAG .
+```
 
-### Tidy up after deploying a hotfix
+1. Push the image to ECR after logging in.
 
-When GitHub is back online, it's time to make Jenkins secure again and merge your changes back into the main repository.
+```
+gds aws govuk-production-poweruser aws ecr get-login-password --region eu-west-1 | docker login --username AWS --password-stdin $ECR_REGISTRY
+docker push $ECR_REGISTRY/$REPO:$IMAGE_TAG
+```
 
-1. Re-enable and run Puppet on the Jenkins instance: `govuk_puppet --enable && govuk_puppet --test`
-1. Restart Jenkins: `sudo service jenkins restart`
-1. Push your hotfix branch to GitHub, open a pull request, and have it reviewed and merged as normal.
-1. This will create a proper release tag, which will automatically deploy through all of the environments if continuous deployment is enabled, or should be manually deployed to all environments if not.
-1. Delete your branch from CodeCommit.
+1. Get the Admin Argo credentials by running `kubectl -n cluster-services get secret argocd-initial-admin-secret -oyaml |yq .data.password |base64 -d`
+1. Log into [Argo](https://argo.eks.integration.govuk.digital/)
+1. Search for the application you wish to deploy in the "Search applications..." search bar
+1. Select the application from the list below the search bar
+1. Select "APP DETAILS"
+1. Go to "DISABLE AUTO-SYNC" then select "ok" when prompted "Disable Auto-Sync?"
+1. Close the "APP DETAILS" menu now select the three dots next to the Deploy object for the part of the application you'd like to redeploy.
+1. Go to the "LIVE MANIFEST" and select "EDIT". find the "image" then update it to the correct image you'd like to deploy. e.g account-api-worker would be for deploying the the account-api-workers.
+1. You can double-check you have the correct image name by doing a `git log` after [checking out the code from CodeCommit](/manual/howto-checkout-and-commit-to-codecommit.html).
```

Comment:
I've merged the above point into the one above it. I'd suggest finishing off with some summary, e.g.

```suggestion
1. Click "Save" and Argo will start the deployment, which should complete in under five minutes (??)
```